amethyst_input/src/axis.rs

```diff
@@ -1,13 +1,26 @@
 use super::Button;
+use super::ControllerAxis;
```

Comment:
Typically we like to group imports in alphabetical order, so this should look like

`use super::{Button, ControllerAxis}`;